uniflow/flow/config.py

```diff
@@ -328,7 +328,6 @@ class RaterForClassificationBedrockClaudeConfig(RaterConfig):
                                                answer, label, and explanation for each case.
     """
 
-    flow_name: str = "RaterFlow"
```

Comment:
qq: is there a reason to remove the default value for `flow_name` while require user to must pass it in?

Reply:
The flow_name has a default value in `RaterConfig`.

Reply:
I also notice that, for transform task, every config has a unique name for different model like `TransformOpenAIFlow`, `TransformHuggingFaceFlow`.

Reply:
You bring a good point, I ended up realizing that we do not need these much classes. We should be able to unified `TrnasformOpenAIFlow` with `TransformHuggingFaceFlow` into a unified `TransformFlow` in the future.